samples/test/automlTranslation.v1beta1.test.js

```diff
@@ -1,129 +0,0 @@
-// Copyright 2018 Google LLC
```

Comment:
If the PR was limited to moving samples from translate over here, it's a little unclear why this is getting deleted

Reply:
I found out most of these old samples were deleted in other languages, and deleted samples' region tags were not used anywhere in codesearch or doc.

note: these tests were not really written, resources and IDs are no where to be found. Tests are very complex. In Python and java, they are already refactored and deleted. Therefore, I deleted and refactored this test into samples/test/automlTranslation.test.js